src/components/confirm/confirm.component.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import I18n from 'i18n-js';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import Dialog from '../dialog/dialog.component';
+import Button from '../button';
+import StyledConfirmButtons from './confirm.style';
+
+class Confirm extends Dialog {
+  /**
+   * Returns main classes for the component combined with
+   * dialog main classes.
+   */
+  get mainClasses() {
+    return classNames(super.mainClasses);
+  }
+
+  /**
+   * Get the yes and no buttons for the confirm dialog
+   *
+   */
+  additionalContent() {
+    return (
+      <StyledConfirmButtons>
+        <Button
+          onClick={ this.props.onConfirm }
+          data-element='confirm'
+          as='primary'
+        >
+          {this.props.confirmLabel || I18n.t('confirm.yes', { defaultValue: 'Yes' })}
+        </Button>
+        <Button
+          onClick={ this.props.onCancel }
+          data-element='cancel'
+          as='secondary'
+        >
+          {this.props.cancelLabel || I18n.t('confirm.no', { defaultValue: 'No' })}
```

Comment:
Could this be set in defaultProps?

Reply:
I guess so but the scope of the ticket was to change the styling only. Changing that part will require changing spec files too so I guess it would be better to leave that for times when the whole component is refactored. What do you think?

Reply:
Makes sense, do we have a separate ticket on the backlog for the refactor?

Reply:
If so I'll mark this as resolved 👍

Reply:
We do - that's the ticket - https://jira.sage.com/browse/FE-1119